src/lib-imap/fuzz-imap-utf7.c

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2020 Dovecot authors, see the included COPYING file */
+
+#include "lib.h"
+#include "str.h"
+#include "fuzzer.h"
+#include "imap-utf7.h"
+
+FUZZ_BEGIN_STR(const char *str)
+{
+	string_t *dest = t_str_new(32);
+
+	imap_utf8_to_utf7(str, dest);
+	imap_utf7_to_utf8(str, dest);
+}
```

Comment:
just want to point out that these { } are not really needed. feel free to resolve =)

Reply:
I think they're still useful. Both for humans and also for editors' auto-indenting.